src/sentry/api/paginator.py

```diff
@@ -45,6 +45,10 @@ class BadPaginationError(Exception):
     pass
 
 
+class MissingPaginationError(Exception):
+    pass
```

Comment:
To add to @sentaur-athena's comment above about the exception, you can setup the string template here and just have the func name, headers, etc passed in